README.md

```diff
@@ -16,7 +16,8 @@ Feel free to add issues in github or send pull requests.
 ### Rules for commits
 
 * Every commit is checked by [Travis CI](https://travis-ci.com/) as soon as
```

Comment:
this is outdated in the meantime same as the first line referencing travis CI

Reply:
It is true but also not related to the change suggested here so I would rather change this in a different PR instead of demanding to extend this one :)

Reply:
I proposed #21 for this